test/e2e/scoped_client_test.go

```diff
@@ -73,12 +73,12 @@ var _ = Describe("Scoped Client", func() {
 				return
 			},
 			assertFunc: func(errGot error) {
-				require.True(GinkgoT(), k8serrors.IsNotFound(errGot))
+				Expect(k8serrors.IsNotFound(errGot)).To(BeTrue())
 			},
 		}),
 	}
 
-	table.DescribeTable("Test", func(tt testParameter) {
+	table.DescribeTable("API call using scoped client", func(testParam testParameter) {
```

Comment:
Would it be simpler and clearer if there were two separate specs instead of being a table with only two entries?

Reply:
Agreed, but I have used `By`s and addressed some of the above review comments which I feel has improved the readability of the test. I am inclined to retain this structure.